### PR TITLE
Addition of contact matrix plot

### PR DESCRIPTION
**Pull request by fsimkovic**

### CODE REVIEW

File: conkit/command_line/conkit_plot.py [hunk 1]
```diff
@@ -144,6 +144,25 @@ def add_contact_map_chord_args(subparsers):
...
+alongside any additional information.
+
+"""
+    subparser = subparsers.add_parser(
+        'cmat',
+        help="Plot a contact map matrix",
+        description=description,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    subparser.add_argument('-e', dest='otherfile', default=None, help='a second contact map to plot for comparison')
+    subparser.add_argument('-ef', dest='otherformat', default=None, help='the format of the second contact map')
```
Comment (hlasimpk) [inline conkit/command_line/conkit_plot.py]: It's an inconsequential detail, but why is the flag for this argument `-e`?

File: conkit/command_line/conkit_plot.py [hunk 2]
```diff
@@ -331,6 +350,24 @@ def altloc_remove(cmap):
...
+        con.sequence = seq
+        con.assign_sequence_register()
+
+        if args.otherfile:
+            other = conkit.io.read(args.otherfile, args.otherformat)[0]
+            other.sequence = seq
+            other.assign_sequence_register()
+        else:
+            other = con
+
```
Comment (hlasimpk) [inline conkit/command_line/conkit_plot.py]: I think you could recode this to:

`if args.otherfile:
    other = conkit.io.read(args.otherfile, args.otherformat)[0]
else:
    other = con

other.sequence = seq
other.assign_sequence_register()`

and remove lines 358 and 359.

Comment (hlasimpk) [reply conkit/command_line/conkit_plot.py]: Actually scratch this, you're plotting them both next to each other so I can see why your format con first.

File: conkit/command_line/conkit_plot.py (hunk 1 above)
Comment (fsimkovic) [reply conkit/command_line/conkit_plot.py]: I use `-o` already for other bits, so `-e` was just a random choice made ages ago

**Pull request merged**